test/functional/ModuleInterfaceTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace RebelCode\Modular\FuncTest\Module;
+
+use RebelCode\Modular\Module\ModuleInterface;
+use Xpmock\TestCase;
+
+/**
+ * Tests {@see RebelCode\Modular\Module\ModuleInterface}.
+ *
+ * @since [*next-version*]
+ */
+class ModuleInterfaceTest extends TestCase
+{
+    /**
+     * The name of the test subject.
+     *
+     * @since [*next-version*]
+     */
+    const TEST_SUBJECT_CLASSNAME = 'RebelCode\\Modular\\Module\\ModuleInterface';
+
+    /**
+     * Creates a new instance of the test subject.
+     *
+     * @since [*next-version*]
+     *
+     * @return ModuleInterface
+     */
+    public function createInstance()
+    {
+        $mock = $this->mock(static::TEST_SUBJECT_CLASSNAME)
+            ->getKey()
+            ->getName()
```

Comment:
The name is, I assume, a nicer version for display purposes. I'm not exactly sure why this is here. To be honest, it might be a mistake.